docs/SOP.md

```diff
@@ -100,6 +101,19 @@ In general, the metadata record of an ontology in the OBO Foundry metadata regis
 
 For a discussion on this SOP, see [here](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1848).
 
+<a name="REACTIVATION"></a>
+
+## Reviving obsolete, orphaned, or inactive ontologies
+
+Ontologies that were marked as either obsolete, orphaned, or inactive require a review process if they want to be marked as active again.
+Currently, these are the required steps:
+1. If ontology reactivation is desired, it must first pass the Dashboard.
+2. A member of the ontology community makes a pull request (PR) on the metadata file with the desired change in status. The PR must include evidence to demonstrate new activity, for example by referring to recently-closed issues (we will provide more detailed guidelines as this SOP matures).
```

Comment:
Is it the case that someone not involved with an ontology makes a PR to reactivate? This seems odd to me.

Reply:
I think what Nico meant was a member of the community OF THAT ONTOLOGY. What would be a better way to make that clear? "A member of the ontology community for the ontology that wants to be reactivated"?

Reply:
I'm not sure that's what he meant, considering the statement that follows regarding the 'ontology owner'.

Reply:
Oh, yeah, you're right. Maybe @matentzn can clarify.

Reply:
Right, the person involved in that ontology who does the PR doesn't need to be the actual ontology owner.

Reply:
Ah, you refer to the contact person? This seems more complicated than necessary, especially considering that the contact person might have moved on and there's a new contact. I'd recommend something like this:

2. A person involved in developing the ontology (usually the contact person) makes a pull request (PR) on the metadata file with the desired change in status. The PR must include evidence to demonstrate new activity, for example by referring to recently-closed issues (we will provide more detailed guidelines as this SOP matures). If the PR was made by someone other than the contact person, or if the contact person has changed, the contact should be tagged in the PR so  that the proper followup can be done. The contact person then has one month to verify that the change is desired.

(delete item 3 as it properly should be merged with 2)

Reply:
That seems reasonable to me. We'll see what @matentzn thinks.

Reply:
Makes sense to me!

Reply:
Can you use suggest mode @nataled to update the phrasing here?

Reply:
@matentzn hopefully I did what you requested. I could not find anything that corresponds to a 'suggest mode', just comment.